js/Repeater/RepeaterComponent.js

```diff
@@ -325,11 +343,20 @@ class Repeater {
     handleEditGroup (repeaterId, event) {
         const edit = this.queryService.get('preview-root')
             .querySelector(`[${this.queryService.getAttr('edit-id')}="${repeaterId}"]`);
+        const $edit = $(edit);
 
```

Comment:
feels odd to have `$edit` & `edit` variables in the same scope here, should we just create the jq object and be done with it?

```
const $edit = $(this.queryService.get('preview-root')
    .querySelector(`[${this.queryService.getAttr('edit-id')}="${repeaterId}"]`));
```

Reply:
Also as an aside, I really regret creating the `queryService`, what on earth was I thinking 🙈